src/main/java/com/plivo/api/models/media/MediaResponse.java

```diff
@@ -1,13 +1,15 @@
 package com.plivo.api.models.media;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.plivo.api.models.base.BaseResponse;
 
 import java.util.List;
 
+//@JsonIgnoreProperties(value = {"status_code","status"})
 public class MediaResponse  extends BaseResponse {
   private List<MediaUploadResponse> objects;
 
-
+  @JsonIgnoreProperties(value = {"status_code","status"})
```

Comment:
@renoldthomas-plivo As you are excluding url and it is mentioned in your test plan doc, do we have to exclude it by including in this annotation?

Reply:
@renoldthomas-plivo ohh nevermind, i see for media upload, fields are just status and status_code